ReactWindows/ReactNative.Net46/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -445,9 +443,9 @@ private void OnLostFocus(object sender, RoutedEventArgs e)
         
         private void OnKeyDown(object sender, KeyEventArgs e)
```

Comment:
>sender [](start = 38, length = 6)

Should this handler be a no-op if this control isn't the `sender`?

Reply:
All the handlers assume (and there's an invariant/assert on JS side) that the text input controls don't support children.

Reply:
All the handlers assume (and there's an invariant/assert on JS side) that the text input controls don't support children.

---
In reply to: [156734920](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156734920) [](ancestors = 156734920)